topiary/tests/samples/expected/toml.toml

```diff
@@ -73,14 +73,13 @@ hosts = [
 cve = [ "CVE-2018-13410", "CVE-2018-1340" ] # garbage CVEs
 comment = "this cve is only valid with attacker-controlled flags to zip"
 
-["unzip"]
-cve = [
-  "CVE-2018-13410",
-  "CVE-2018-1340"
+["inline.comments"]
```

Comment:
I think we should add the `inline.comments` section alongside `unzip`, instead of replacing it

Reply:
I didn't notice that you were the original author of this section ; my bad.